src/trio/testing/_raises_group.py

```diff
@@ -503,6 +605,25 @@ def __enter__(self) -> ExceptionInfo[BaseExceptionGroup[BaseException]]:
         )
         return self.excinfo
 
+    def __str__(self) -> str:
+        return self.__repr__()
+
+    def __repr__(self) -> str:
+        reqs = [
```

Comment:
```suggestion
        parameters = [
```

I don't really understand what `reqs` means here.